pkg/utils/activity/event.go

```diff
@@ -78,6 +78,7 @@ const (
 	LookupActivity
 	SearchActivity
 	RegisterActivity
+	FetchKeyActivity
```

Comment:
Mind if we call this `KeyExchangeActivity` to match better with the TRISA terminology?

Reply:
makes sense - I have made the update!